pyproject.toml

```diff
@@ -70,7 +72,9 @@ build_command = "pip install poetry && poetry build"
 
 [tool.poetry.dependencies]
 python = "^3.8"
```

Comment:
Looking at it, `wemake-python-styleguide` should be upgraded to `0.18.0` as it'll pull in the updated version of bandit once released, but needs python to be `3.8.1`.

Reply:
Not using ruff yet?

Reply:
Funny you say that, as that's 100% on my ToDo. I started a working branch a while ago, but yes, I absolutely should.